docroot/modules/custom/sitenow_articles/src/Plugin/Block/PreservedLinksMessage.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\sitenow_articles\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * A preserved links message block.
+ *
+ * @Block(
+ *   id = "preservedlinksmessage_block",
+ *   admin_label = @Translation("Preserved Links Message Block"),
+ *   category = @Translation("Restricted")
+ * )
+ */
+class PreservedLinksMessage extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return ['label_display' => FALSE];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $markup = '<div>Links in this article are preserved for historical purposes, but the destination sources may have changed.</div>';
+
+    return [
+      '#markup' => $markup,
+      '#attributes' => [
+        'class' => [
+          'alert alert-info',
+        ],
+      ],
+    ];
```

Comment:
Can we move the `alert` and `alert-info` into separate, individual entries in the `class` array? I think it should work either way, but just for consistency.

I also think we could drop the `<div>...</div>` around the `$markup` message, because the block will be rendered in a div with the 'alert alert-info' classes.